docs/use-cases/sunhwi_use_case.md

```diff
@@ -0,0 +1,5 @@
+## use-case for post measurement
```

Comment:
Sun Hwil, please take a look at Mike's UC to see how to write these.

I like what you are trying to do, but it is helpful when we come to code it if we have a rather specific and detailed account of who is trying to do what (and why).
target audience -> are we coding for a skilled knowledgable python person or an african undergrad, or who?  Identify a real person that fits the right profile nad use their name (it makes it very real!)

Read: what do you want to "show" and how.  Not implementational details, but what does "person" want to see and how does she want to interact with it?  What does she hope to learn from it?

Same for the other bullets.  In fact, each of your bullets should be one or two different UCs.